### ISSUE TITLE
More responsive dictionary/perspective renaming

### ISSUE THREAD

**Issue by myrix** [ispras/lingvodoc-react#825]
Open any dictionary, e.g. http://83.149.198.62/dictionary/3648/5/perspective/3648/8/edit, logged in as admin, open perspective properties,
![image](https://user-images.githubusercontent.com/2802467/174450496-0a1a61eb-735b-4108-b831-2567f0e4db44.png)
rename perspective,
![image](https://user-images.githubusercontent.com/2802467/174450558-0bc4aa37-0c30-4c58-b3be-1d17c462ce0c.png)
close properties dialog, the name does not change:
![image](https://user-images.githubusercontent.com/2802467/174450579-9ad2a890-b180-41ec-8673-c0b79051fcc2.png)

The same with dictionary renaming:
![image](https://user-images.githubusercontent.com/2802467/174450630-3a0cfb8a-7326-4bed-83f7-481735d85e00.png)
![image](https://user-images.githubusercontent.com/2802467/174450659-4a130a29-4a4e-4c10-b5f7-ab2dd246234d.png)
![image](https://user-images.githubusercontent.com/2802467/174450662-179d36d6-e723-4e26-94b7-346c5fc5a2c2.png)

Name changes become apparent only after page reload:
![image](https://user-images.githubusercontent.com/2802467/174450692-0ba9e4b3-a9f6-4061-9d5b-44ecdc1d759d.png)

If possible, new names should be shown immediately or soon after names are changed in properties so there is less possible confusion for users.

**Comment by jandrew44** [comment]
Duplicate of #379.

**Comment by vmonakhov** [comment]
Covered in tesst/test_dictionaries.py
